### DISCUSSION TITLE
WARNING: mseide-msegui mailing list is no more accessible.

### DISCUSSION THREAD

**Discussion by fredvs** [mse-org/mseide-msegui#57] [General]
Hello.

There are problems with sourceforge mailing-list server.
Med and I try to sent mails but they do not appear.

Maybe it would be better to use this issue-github-site for question about msegui code.

Fre;D

**Comment by fredvs** [comment]
Hello!

Today, in the forum-mailing list http://mseide-msegui-talk.13964.n8.nabble.com/ ,
all the mails of Med, Sieghard, Graeme, Code DZ, Santiago and others were deleted "by the author".
I suppose it was not the case but something else happens.

If it was the case, it is even more strange.

Fre;D

**Comment by graemeg** [comment]
Nope, I didn't delete anything.

**Comment by lainz** [comment]
Strange enough. Like my posts being deleted in the Lazarus.

Two connected universes with same problems.

**Comment by fredvs** [comment]
uos forum is also infected now.

What a strange time we live in! ...

;-(

**Comment by fredvs** [comment]
Ooops and all the mails of fpc nabble mailing list too!

http://free-pascal-general.1045716.n5.nabble.com/

![fpcnable](https://user-images.githubusercontent.com/3421249/124117352-5314da80-da70-11eb-85ac-94f42d1bbd0e.png)

**Comment by fredvs** [comment]
Hello.I think it is a problem of Nabble.

I prefer this.

Fre;D

**Comment by fredvs** [comment]
Here the issue:
http://support.nabble.com/CONTENTS-DELETED-td7609604.html

**Comment by lainz** [comment]
Indeed the logic first.

**Comment by fredvs** [comment]
Yes and before the logic, should be the respect.

Nabble could warn that they will not host mailing list anymore.
And not delete randomly some users and write "Deleted by the author".

But yes, the logic of the respect is sometime strange.
